src/libs/ui/src/organisms/Header/index.tsx

```diff
@@ -108,32 +109,93 @@ const Header: React.FC<headerProps> = ({ modalSetToggle, handleRegisterClick })
   };
 
   const handleNavLinkClick = (title: string) => {
-    const catagory =  appSetting.find(link => link.title === selectNavLink);
-    const isNavigate = catagory && catagory?.canNavigate && catagory?.navigationPages.includes(title);
-    // const selectedLink = appSetting.find(link => link.title === selectNavLink)?.canNavigate;
-  
+    console.log('title', title, selectNavLink)
+    const catagory = appSetting.find(link => link.title.toLocaleLowerCase() === selectNavLink.toLocaleLowerCase());
+    console.log('encodedTitle', catagory)
+    const isNavigate = catagory && catagory?.canNavigate && (
+      catagory?.navigationPages.includes(title)
+      || selectNavLink === appSetting[0].title
+    );
     if (isNavigate) {
-      console.log('url',title)
-      const encodedTitle =  title.replace(/[\s&]+/g, '-');
-      navigate(`${selectNavLink}/${encodedTitle}`);
+      const encodedTitle = title.replace(/[\s&]+/g, '-');
+      if (selectNavLink === appSetting[3].title) {
+        navigate(`${selectNavLink}/${encodedTitle}`);
+      }
+
+      // const searchParams = new URLSearchParams(location.search); // Create a new instance of URLSearchParams
+
+      // Create an array of navigation links with their corresponding query parameters
+      const navLinkMap = [
+        { title: appSetting[0].title, category: encodedTitle },
```

Comment:
if appSetting[0] is undefined then we will get error, please check this condition